scripts/mutatenodes.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -e
+
+SEND_APP_NAME='malleusSender'
+RECV_APP_NAME='malleusReceiver'
+
+usage() {
+    echo "Usage: $0 --hub-url=HUB_URL --num-senders=NUM_SENDERS --send-node-max-sessions=SEND_NODE_MAX_SESSIONS --recv-node-max-sessions=RECV_NODE_MAX_SESSIONS" >&2
+    exit 1
+}
+
+# new arg parsing code that includes default values for the different options.
+for arg in "$@"; do
+    optname=`echo $arg | cut -d= -f1`
+    optvalue=`echo $arg | cut -d= -f2`
+    case $optname in
+	--hub-url) HUB_URL=$optvalue;;
+	--num-senders) NUM_SENDERS=$optvalue;;
+	--send-node-max-sessions) SEND_NODE_MAX_SESSIONS=$optvalue;;
+	--recv-node-max-sessions) RECV_NODE_MAX_SESSIONS=$optvalue;;
+	*)
+	    usage
+	    ;;
+    esac
+done
+
+if [ -z "$HUB_URL" -o -z "$NUM_SENDERS" -o -z "$SEND_NODE_MAX_SESSIONS" -o -z "$RECV_NODE_MAX_SESSIONS" ]
+then
+    usage
+fi
+
+if [ -z "$JITSI_SSH_CONFIG" ]
+then
+    JITSI_SSH_CONFIG="$HOME/.ssh/config.d/jitsi64.config"
+fi
+
+if [ -z "$SSH" ]
+then
+    if [ -r "$JITSI_SSH_CONFIG" ]
+    then
+	SSH="ssh -F $JITSI_SSH_CONFIG"
+    else
+	SSH="ssh"
+    fi
+fi
+
+if [ -z "$SCP" ]
+then
+    if [ -r "$JITSI_SSH_CONFIG" ]
+    then
+	SCP="scp -F $JITSI_SSH_CONFIG"
```

Comment:
I think there are tabs here? Maybe you can use 4 spaces instead (just for consistency with the rest of the file)

Reply:
hmm it seems the file has a mixture of 4 spaces, 2 spaces and tabs? I don't have any preference, but maybe it's worth choosing one spacing style and apply it across all lines

Reply:
Fixed.